goc/share/generate_sshauthkeys

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
+
+import sys
+import os
+import pwd, grp
+import re
+import subprocess
+import json
+
+# it's much easier to switch to curl for the authenticated lookup than
+#  to use native python
+cmd = 'curl -s --cert /etc/grid-security/hostcert.pem --key /etc/grid-security/hostkey.pem https://topology.opensciencegrid.org/oasis-managers/json?vo="*"'
+p = subprocess.Popen( ('/bin/bash', '-cx', cmd), bufsize=1, stdout=subprocess.PIPE)
```

Comment:
Just curious, why the `bufsize=1` ?

Reply:
Thanks for asking, because it was a copy/paste error from another application.  I will delete that.